test/libsolidity/SemanticTest.cpp

```diff
@@ -52,6 +52,28 @@ SemanticTest::SemanticTest(string const& _filename, langutil::EVMVersion _evmVer
 	m_lineOffset(m_reader.lineNumber()),
 	m_enforceViaYul(enforceViaYul)
 {
+	auto simpleSmokeBuiltin = [](FunctionCall const& _call) -> std::optional<bytes> {
+		// This function is only used in test/libsolidity/semanticTests/builtins/smoke.sol.
+		// It could be removed when we have actual builtins.
+		std::optional<bytes> result;
+		if (_call.arguments.parameters.size() < 3)
+		{
+			result = bytes();
+			for (auto const& parameter: _call.arguments.parameters)
+				result.value() += util::toBigEndian(u256{util::fromHex(parameter.rawString)});
+		}
+		return result;
+	};
```

Comment:
iiuc, this built in sums up the parameters of a function call with fewer than 3 parameters?